cl/opinion_page/templates/court.html

```diff
@@ -86,16 +107,34 @@ <h3>Recent Decisions of the&nbsp;{{ court }} <i class="gray fa fa-file-text-o"><
           <p>No cases yet.</p>
         {% endif %}
       </div>
-      <div class="col-xs-4">
-        <h3>About the Court</h3>
-          <p>The Supreme Judicial Court of Maine is the highest court in the state of Maine. It is made up of a chief justice and six associate justices who are appointed by the governor and confirmed by the Maine Senate. The court has general appellate jurisdiction and is responsible for interpreting the state's laws and constitution. It also has the power to issue advisory opinions on matters of law to the governor and the state's legislature.</p>
-        <hr>
-        <h3>Neutral Citations</h3>
-          <p>Supreme Judicial Court of Maine court uses neutral citations to cite its opinions.</p>
-        <p>Citations to the {{ court }} take the form:</p>
-        <pre>[YEAR] ME [#]</pre>
-        <p>In this cases, the format begins with the year, then an abbreviation for the court (ME), and finishes with a serial number indicating the decision number.</p>
-      </div>
+      {% if pk == "me" %}
+        <div class="col-md-4">
+          <h3>About the Court</h3>
+          <p>The Supreme Judicial Court of Maine is the highest court in the
+            state
+            of Maine. It is made up of a chief justice and six associate
+            justices who are appointed by the governor and confirmed by the
+            Maine Senate. The court has general appellate jurisdiction and
+            is
+            responsible for interpreting the state's laws and constitution.
+            It
+            also has the power to issue advisory opinions on matters of law
+            to
+            the governor and the state's legislature.</p>
+          <hr>
+          <h3>Neutral Citations</h3>
+          <p>Supreme Judicial Court of Maine court uses neutral citations to
+            cite
+            its opinions.</p>
+          <p>Citations to the {{ court }} take the form:</p>
+          <pre>[YEAR] ME [#]</pre>
+          <p>In this cases, the format begins with the year, then an
+            abbreviation
+            for the court (ME), and finishes with a serial number indicating
+            the
+            decision number.</p>
+        </div>
```

Comment:
some where whitespacing going on around here.

Reply:
i have already adjusted some of the white spaces to reduce the number of changes, there were several changes to the template and pycharm formatted it differently but i think it is because indent_size setting in .editorconfig file

Reply:
thanks for fixing that